WorkflowReactiveSwift/Sources/SignalProducerWorkflow.swift

```diff
@@ -47,18 +47,27 @@ struct SignalProducerWorkflow<Value>: Workflow {
     public typealias State = Void
     public typealias Rendering = Void
 
+    struct Action: WorkflowAction {
```

Comment:
this seems like a good change to improve the utility of type-level introspection. do you think repeating the associated workflow's type in the action's type itself would be valuable? i.e. `SignalProducerWorkflowAction` or something. or putting that info into the string via 'custom string convertible' or whatever the appropriate conformance may be.

Reply:
I did consider that yeah. I'll go ahead and make that change.

I also considered pulling a generic `SendOutputAction` out as a standalone type, but wasn't sure if that was worth it